src/styles/stars.scss

```diff
@@ -0,0 +1,27 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+  width: 200px;
+
+  &-list {
+    display: flex;
+  }
+}
+
+.card__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+
+  &:not(:last-child) {
+    padding-right: 4px;
+  }
+
+  &--active:not(:last-child) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
The use of the BEM naming convention seems inconsistent here. The modifier '--active' should be applied to the block or element it modifies, such as '.card__star--active'. Ensure that the BEM naming convention is consistently applied throughout your styles.